.github/workflows/autoformat.yml

```diff
@@ -181,9 +184,12 @@ jobs:
           path: /tmp
 
       - name: Apply patch and push
+        env:
+          RUN_ID: ${{ github.run_id }}
```

Comment:
What kind of attacker scenario are we caring about here?

Reply:
Good question. We can actually trust `github.run_id` because it's generated by GitHub. In general, using `${{ ... }}` is unsafe because that would allow an attacker to run arbitrary code.

Ref: https://securitylab.github.com/resources/github-actions-untrusted-input/

Reply:
So the original code looks **unsafe**, but the new code looks safer.

Reply:
Yeah, I understand that ref might be unsafe and wonder how users can attach "github.run_id". If this is just in case, then agree with the intention!

Reply:
I'm not a big fan of `looks unsafe` because I need to check if's safe or not :)